Berksfile

```diff
@@ -32,35 +32,32 @@ cookbook 'kafka-bcpc', path: './cookbooks/kafka-bcpc'
 #
 cookbook 'apt'
 cookbook 'chef-client'
-cookbook 'chef-ingredient'
 cookbook 'java'
 cookbook 'maven'
 cookbook 'ntp'
 cookbook 'pam'
 cookbook 'ubuntu'
 
 #
-# Cookbooks absent from the supermarket
+# Transitive dependencies that have been forked for one reason or another.
 #
 
 # bfd is not on the supermarket
 cookbook 'bfd',
   git: 'https://github.com/bloomberg/openbfdd-cookbook'
 
+
 # cobblerd forked, pending destruction of earth by moon.
-cookbook 'cobblerd',
-  git: 'https://github.com/cbaenziger/cobbler-cookbook',
-  branch: 'cobbler_profile'
+cookbook 'cobblerd', 
+  git: 'https://github.com/bloomberg/cobbler-cookbook'
```

Comment:
Gah! Thanks for pointing this to the Bloomberg version; I'm happy I haven't accidentally broken us this long.